python-large-file-upload/main.py

```diff
@@ -41,7 +42,7 @@ def parseCommandLineArgs():
     p.add("-i", "--include-file-name", help="Add file name as query parameter to web request", action='store_true')
     p.add("-e", "--measure-elapsed-time", help="Measure how long the file transfer process takes", action='store_true')
     p.add("--legacy", help="Use legacy method to upload file. Uses base64 encoding in web transaction payloads", action='store_true')
-    
+    p.add("-w", "--web-req-type", help="HTTP request type (PUT, POST, etc)", default="POST")

```

Comment:
The official name is a `request method`

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

Since this is a published example, I recommend changing it from `--web-req-type` to `--web-req-method`.  And update the `help` field to refer to it as `HTTP request method`